lib/crowbar/init/application.rb

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+require "/opt/dell/crowbar_framework/lib/crowbar/upgrade_status"
```

Comment:
I agree with @aspiers we should not hardcode the complete path. Not sure if this is the right approach but we can add the lib folder to the $LOAD_PATH variable.

See http://stackoverflow.com/questions/9474299/what-are-the-paths-that-require-looks-up-by-default